androidviewertemplates/src/main/java/com/example/evan/androidviewertemplates/firebase_classes/TeamTemplate.java

```diff
@@ -11,51 +11,9 @@
 /**
  * Created by Teo on 1/10/2018.
  */
-@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonIgnoreProperties(ignoreUnknown = false)
```

Comment:
Got it.  Is that something that could be separated into a different PR?

Reply:
No, they'll be changed once in a while just to check schema. For the moment it should be off.

Reply:
Ah okay.  Will they only be changed for testing? If so, `master` should always have them off, right?

Also, should they be "off" regardless of if there's 2018 schema?

Reply:
Ok cool, so it's only for testing and should be `false` on `master`.  Could you separate that into a different PR since that's not directly related with removing 2018 schema?

Reply:
Shouldn't I leave it, since it has to do with schema, and should be off when schema is first added?

Reply:
It isn't related to removing old schema, and should be separated into a different PR.